Library/Homebrew/cmd/reinstall.rb

```diff
@@ -43,6 +39,9 @@ def reinstall_formula(f)
     fi.verbose             = ARGV.verbose?
     fi.debug               = ARGV.debug?
     fi.prelude
+
+    oh1 "Reinstalling #{f.full_name} #{options.to_a.join " "}"
```

Comment:
This will have a trailing space in the case of no options, which is not the case with the current code.
```
iMac-TMP:homebrew-core joe$ brew reinstall ffmpeg$
==> Reinstalling ffmpeg $
==> Downloading https://homebrew.bintray.com/bottles/ffmpeg-3.2.2.el_capitan.bottle.tar.gz$
Already downloaded: /Users/joe/Library/Caches/Homebrew/ffmpeg-3.2.2.el_capitan.bottle.tar.gz$
==> Pouring ffmpeg-3.2.2.el_capitan.bottle.tar.gz$
🍺  /usr/local/Cellar/ffmpeg/3.2.2: 239 files, 50.4M$
```